xml/System.Xml/XmlDocumentXPathExtensions.xml

```diff
@@ -68,9 +68,9 @@
         <Parameter Name="node" Type="System.Xml.XmlNode" RefType="this" />
       </Parameters>
       <Docs>
-        <param name="node">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="node">The <see cref="T:System.Xml.XmlNode" /> you want the navigator initially positioned on.</param>
+        <summary>Creates an <see cref="T:System.Xml.XPath.XPathNavigator" /> for navigating this object.</summary>
+        <returns>An <see langword="XPathNavigator" /> object used to navigate the node. The <see langword="XPathNavigator" /> is positioned on the node from which the method was called. It is not positioned on the root of the document.</returns>
```

Comment:
Some natural language suggestions:
```suggestion
        <param name="node">The node where the navigator is initially positioned.</param>
        <summary>Creates an XPath navigator for navigating the specified node.</summary>
        <returns>An XPath navigator object used to navigate the node. The XPath navigator is positioned on the node from which the method is called, not on the root of the document.</returns>
```

Reply:
except for `this` arg all of those are copy paste of existing docs